Utils/Dataflow/069_upload2es/load_data.sh

```diff
@@ -57,23 +99,34 @@ load_files () {
   do
     ${cmd}${INPUTFILE} || exit 3
   done
+  echo -ne "$EOProcess"
 }
 
 load_stream () {
   log "Switched to the stream mode."
-  while read -r -d "$DELIMITER" line; do
+  while read -r -d $(echo -ne "$EOBatch") line; do
```

Comment:
Still not working with "space"-like characters, including the newline character...
When we use `$(command)`, the space-like characters (newline, zero byte, etc) are stripped form the `command` output. We discussed this fact many times but I never noticed that here it is the same case...

I apologize for the mislead.

Seems like there\`s no other way but passing the already interpreted EOBatch symbol to the script, so that the stage is run like this:
```
  ./load_data.sh -E $'\n' -B $'\0'
```

Here is a modelled situation to check that it works:
```
$ f() {
>   EOBatch="$1"
>   while read -r -d "$EOBatch" line; do
>     echo "line: '$line'"
>   done
> }
$ echo -ne 'First line\x00Second line\x00' | f $'\x00'
line: 'First line'
line: 'Second line'
$
$ echo 'First line
Second line' | f $'\n'
line: 'First line'
line: 'Second line'
```

There may be a way to achieve what we need by `read`ing interpreted value from `echo`, but for now I do not see how to do it properly (simple pipe won\`t work). And again -- we can not write `\x00` into a variable.

---

Passing of "encoded" strings (like `'\n'` instead of `$'\n'` or `'

'`) is an appropriate option for markers used only in the output (like EOP); but for those used to interpret input, we want to pass them to some Shell command as parameters, and thus need to deal with the interpreted value... which is not a proper task for Shell, actually.